test/integration.test.js

```diff
@@ -263,7 +264,8 @@ describe('lint-staged', () => {
     const status = await execGit(['status'])
     expect(status).toMatch('modified:   test.js')
     expect(status).toMatch('no changes added to commit')
-    expect(await readFile('test.js')).toEqual(testJsFilePretty + appended)
+    /** @todo `git` in GitHub Windows runners seem to add `\r\n` newlines in this case. */
+    expect(normalizeWindowsNewlines(await readFile('test.js'))).toEqual(testJsFilePretty + appended)
```

Comment:
For whatever reason in this case on Windows the files need to be normalized by converting `\r\n` to `\n`... Workaround seems to fix it.